Sources/LiveViewNative/Modifiers/DisabledModifier.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DisabledModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 3/29/23.
+//
+
+import SwiftUI
+
+/// Disables interaction for any element.
+///
+/// ```html
+/// <Button
+///     modifiers={
+///         disabled(@native, disabled: true)
+///     }
+/// >
+///  <Label>This button is disabled</Label>
+/// </Button>
+/// ```
+///
+/// ## Arguments
+/// * ``disabled``
+struct DisabledModifier: ViewModifier, Decodable, Equatable {
+    private let disabled: Bool?
```

Comment:
This needs the `@_documentation(visibility:)` attribute for the symbol link to work.
```suggestion
    #if swift(>=5.8)
    @_documentation(visibility: public)
    #endif
    private let disabled: Bool
```